package.json

```diff
@@ -23,6 +23,7 @@
     "morgan": "~1.9.1",
     "moment": "^2.24.0",
     "msgpack5": "^4.2.1",
+    "node-jose": "^1.1.3",
```

Comment:
Just curious as to what this new library adds/does?

Reply:
This library is for encrypting and signing a jwt, which we need for sending the 'state' paramater to the account service for authentication. 
This library is written by cisco, and recommended by jwt.io for a js implementation of jwt encryption.